packages/go/safety/fixtures_test.go

```
@@ -0,0 +1,278 @@
+		"我今天想跟你说一件事，" + base + "，好不好",
+		zeroWidthSpread(base),
+	}
+}
+
+// zeroWidthSpread 每两个 rune 之间插入零宽空格（伪装变体；normalize 剥离后还原）。
+func zeroWidthSpread(s string) string {
+	var b strings.Builder
+	runes := []rune(s)
+	for i, r := range runes {
+		if i > 0 {
+			b.WriteString("​")
```

golangci-lint (2.12.2)

[error] ST1018: string literal contains the Unicode format character U+200B, consider using the '\u200b' escape sequence instead

(staticcheck)